src/tests/setup.ts

```
@@ -0,0 +1,38 @@
+    media: query,
+    onchange: null,
+    addListener: () => {},
+    removeListener: () => {},
+    addEventListener: () => {},
+    removeEventListener: () => {},
+    dispatchEvent: () => {},
+  }),
+});
+
+// Mock leaflet since it doesn't work well in jsdom
+vi.mock('leaflet', () => ({
```

ESLint

[error] 'vi' is not defined.

(no-undef)